src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -331,16 +335,15 @@ def load_pytorch_state_dict_in_tf2_model(
                 )
                 raise tf.errors.InvalidArgumentError(error_msg)
             else:
-                mismatched_keys.append((name, pt_state_dict[name].shape, symbolic_weight.shape))
+                mismatched_keys.append((name, array, symbolic_weight.shape))
```

Comment:
Is is ok to change type here ? Shouldn't it be `array.shape`?

Reply:
Oh woah, this is a bug, you're right. Sorry!

Reply:
Good catch, there is `shape` missing here.

Reply:
Fixed!